themes/default/content/blog/building-a-data-warehouse-on-aws-with-redshift-and-pulumi/index.md

```diff
@@ -457,6 +457,10 @@ SELECT *
 
 Huzzah! You've now got yourself a working data warehouse.
 
+Here's a [Modern Infrastructure](https://www.youtube.com/playlist?list=PLyy8Vx2ZoWloyj3V5gXzPraiKStO2GGZw) video that demonstrates everything you've just read in this blog post:
```

Comment:
can we put video into the link text?

Reply:
@susanev For sure! Would you like to replicate that change to the other blog posts in this PR as well?

Reply:
yah! wasnt sure if it was easier for you for me to repeat a suggested change. would you like me to?

Reply:
Nah, no need. I'll make the same change elsewhere.